Updating secret management page (#7387)
umbraco/UmbracoDocs: umbraco-cloud/begin-your-cloud-journey/project-features/secrets-management.md

## How to add secrets

{% hint style="warning" %}

## Important

When adding a secret to your environment, it will restart.

Secrets are stored as environment variables.
The underlying platform has a maximum size limit for all environment variables combined.
If too many secrets are added, or if secret values are too large, your environment may fail to start.

> We Recommend:
> 
> - Keeping secrets small and concise.
> - Storing only sensitive values as secrets (for example: API keys, connection strings).
> - Using appsettings.json for general configuration values.

Reviewer: ```suggestion
It is recommended to:

- Keep secrets small and concise.
- Store only sensitive values as secrets (for example: API keys, connection strings).
- Use `appsettings.json` for general configuration values.
```
